packages/types/ContactJob.ts

```diff
@@ -40,6 +40,11 @@ export type PublishToContactJobsTopicParams = PublishRetrieveContactTranscript;
 
 //====== Message payloads expected for the completed contact jobs ======//
 
+type CompleteRetrieveContactTranscriptTSuccess = {
```

Comment:
typo? should it be `CompleteRetrieveContactTranscriptTSuccess`?

Reply:
The `T` prefix is just a convention when naming generic type parameters, but the types you provide when you instantiate those generic types should not be named such a way.

e.g. 

```
type CompletedContactJobMessageCommons<TSuccess, TFailure> = { ... } // T prefix on the type generic parameters
const myCompletedContactJobMessage: CompletedContactJobMessageCommons<MySuccessImplementation, MyFailureImplementation> = { ... } // No T prefix on the types we provide for TSuccess or TFailure when crwating an instance of CompletedContactJobMessageCommons
```